src/System.Drawing.Common/tests/ImageTests.cs

```diff
@@ -103,5 +105,83 @@ public void FromStream_NullStream_ThrowsArgumentNullException()
+        [InlineData(PixelFormat.Format4bppIndexed, 4)]
+        [InlineData(PixelFormat.Format8bppIndexed, 8)]
+        [InlineData(PixelFormat.Format16bppArgb1555, 16)]
+        [InlineData(PixelFormat.Format16bppGrayScale, 16)]
+        [InlineData(PixelFormat.Format16bppRgb555, 16)]
+        [InlineData(PixelFormat.Format16bppRgb565, 16)]
+        [InlineData(PixelFormat.Format24bppRgb, 24)]
+        [InlineData(PixelFormat.Format32bppArgb, 32)]
+        [InlineData(PixelFormat.Format32bppPArgb, 32)]
+        [InlineData(PixelFormat.Format32bppRgb, 32)]
+        [InlineData(PixelFormat.Format48bppRgb, 48)]
+        [InlineData(PixelFormat.Format64bppArgb, 64)]
+        [InlineData(PixelFormat.Format64bppPArgb, 64)]
+        public void GetPixelFormatSize_ReturnsExpected(PixelFormat format, int expectedSize)
+        {
+            Assert.Equal(expectedSize, Image.GetPixelFormatSize(format));
+        }
+
+        [Theory]
+        [InlineData(PixelFormat.Format16bppArgb1555, true)]
+        [InlineData(PixelFormat.Format32bppArgb, true)]
+        [InlineData(PixelFormat.Format32bppPArgb, true)]
+        [InlineData(PixelFormat.Format64bppArgb, true)]
+        [InlineData(PixelFormat.Format64bppPArgb, true)]
+        [InlineData(PixelFormat.Format16bppGrayScale, false)]
+        [InlineData(PixelFormat.Format16bppRgb555, false)]
+        [InlineData(PixelFormat.Format16bppRgb565, false)]
+        [InlineData(PixelFormat.Format1bppIndexed, false)]
+        [InlineData(PixelFormat.Format24bppRgb, false)]
+        [InlineData(PixelFormat.Format32bppRgb, false)]
+        [InlineData(PixelFormat.Format48bppRgb, false)]
+        [InlineData(PixelFormat.Format4bppIndexed, false)]
+        [InlineData(PixelFormat.Format8bppIndexed, false)]
+        public void IsAlphaPixelFormat_ReturnsExpected(PixelFormat format, bool expected)
+        {
+            Assert.Equal(expected, Image.IsAlphaPixelFormat(format));
+        }
+
+        public static IEnumerable<object[]> GetEncoderParameterList_ReturnsExpected_TestData()
+        {
+            yield return new object[]
+            {
+                ImageFormat.Tiff,
+                new Guid[]
+                {
+                    Encoder.Compression.Guid,
+                    Encoder.ColorDepth.Guid,
+                    Encoder.SaveFlag.Guid,
+                    new Guid(unchecked((int)0xa219bbc9), unchecked((short)0x0a9d), unchecked((short)0x4005), new byte[] { 0xa3, 0xee, 0x3a, 0x42, 0x1b, 0x8b, 0xb0, 0x6c }) /* Encoder.SaveAsCmyk.Guid */
+                }
+            };
+        }
+
+        [ActiveIssue(20884, TestPlatforms.AnyUnix)]
+        [ConditionalTheory(Helpers.IsDrawingSupported)]
+        [MemberData(nameof(GetEncoderParameterList_ReturnsExpected_TestData))]
+        public void GetEncoderParameterList_ReturnsExpected(ImageFormat format, Guid[] expectedParameters)
+        {
+            if (PlatformDetection.IsFullFramework)
+            {
+                return;
```

Comment:
Since we're using `ConditionalTheory` you can throw a `SkipTestException` and it will be marked as skipped instead of silently skipping it.

Reply:
OK, done. I also re-enabled a couple of `FromFile` tests, since that code is now shared across Windows & Unix, let's see what CI thinks about that.